docs/source/getting_started/getting_started.md

```diff
@@ -0,0 +1,226 @@
+    -H "authorizationToken: <front_office_token>"
+```
+
+As a result, we will get something like:
+
+```json
+{
+  "documents": [
+    {
+      "id": "xayn_5283ef3",
+      "score": 0.8736,
+      "properties": {
+        "title": "Why every bit matters",
+        "link": "https://www.xayn.com/blog/why-every-bit-matters",
+        "image": "https://uploads-ssl.webflow.com/5ef08ebd35ddb63551189655/61447d6ebda40f1487c6ed9a_noah-silliman-2ckQ4BrvpC4-unsplash-p-2000.jpeg"
+      }
+    },
+    {}
+  ]
+}
+```
+
+In the request, we ask the system to include the properties of the returned documents. We can use this data to implement a 'more like this' section.
+
+We also have a 'score' field which represents how well the documents match the user's interests. The higher the number, the better the documents match.
+
+# Search
+
+Depending on the use-case searching for documents can be achieved as a search for documents _similar_ to a given document or as a _free-text search_. Both variants can then be run as a anonymous search or a search that is personalized. Personalization comes in two fashions, with a _user-id_ or by providing a interaction _history_.
+
+## Similar documents
+
+In this search variant only a _document id_ must be provided to the [`/semantic_search`](https://docs.xayn.com/front_office.html#tag/front-office/operation/getSimilarDocuments) endpoint.
+
+```bash
+curl -X POST https://<url>/semantic_search
+    -H "authorizationToken: <front_office_token>"
+    -H "Content-Type: application/json"
+    -d '{
+        "document": { "id": "xayn_cd5604c" }
+    }'
+```
+
+The result contains a list of documents that are similar to the `snippet` of the provided _document id_.
+
+## Free Text search
+
+Just like [Similar documents](#similar-documents) it is also possible to run a free text search.
+
+```bash
+curl -X POST https://<url>/semantic_search
+    -H "authorizationToken: <front_office_token>"
+    -H "Content-Type: application/json"
+    -d '{
+        "document": { "query": "Privacy and security" }
+    }'
+```
+
+The quality of the results can vary on the length of the provided query. Short queries usually yield better results with the [hybrid search option](https://docs.xayn.com/front_office.html#tag/front-office/operation/getSimilarDocuments) enabled, that combines semantic and bm25 search:
+
+```json
+{
+  "document": { "query": "Privacy and security" },
+  "enable_hybrid_search": true
+}
+```
+
+## Personalised search
+
+Any search can also be combined with an `user id` or a user history, which is a list of `interactions`.
+To personalise search results for a specific user, any search can also be combined with an `user id` or a user `history`, which is a list of interactions of a particular user. The option to use a user history of interactions instead of a user id enables a personalised search without the need for Xayn to store a user id or history of interactions.
```

Comment:
:art:
This sentence is a repetition of the above.
`any search can also be combined with a `user id` or a user `history`, which is a list of interactions of a particular user`